frontend/src/component/PublicProfile/CustomFormManager.jsx

```diff
@@ -0,0 +1,289 @@
+    </div>
+  )
+}
+
+const DisplaySchemaPropertyString = props => {
+  return (
+    <div className='DisplaySchemaPropertyString'>
+      <span
+        className={classnames(
+          'DisplaySchemaPropertyString__label',
+          { noLabel: !props.label }
+        )}
+      >
+        {props.label}
+      </span>
+      <span className='DisplaySchemaPropertyString__value'>{props.value}</span>
+    </div>
+  )
+}
+
+const DisplaySchemaPropertyBoolean = props => {
+  return (
+    <div className='DisplaySchemaPropertyString'>
+      <span
+        className={classnames(
+          'DisplaySchemaPropertyString__label',
+          { noLabel: !props.label }
+        )}
+      >
+        {props.label}
+      </span>
+      <span className='DisplaySchemaPropertyString__value'>
+        <Checkbox
+          name={props.label}
+          checked={props.value}
+          onClickCheckbox={() => {}}
+          styleLabel={{ top: '8px', cursor: 'default' }}
+          styleCheck={{ top: '-5px' }}
+        />
+      </span>
+    </div>
+  )
+}
+
+const DisplaySchemaArray = props => {
+  if (!props.valueList) return null
+
+  if (props.valueList.every(value => typeof value === 'string')) {
+    return (
+      <DisplaySchemaPropertyString
+        label={props.label}
+        value={props.valueList.join(', ')}
+      />
+    )
+  }
+
+  return [
+    <DisplayTitle
+      label={props.label}
+      key={props.label}
+    />,
+    props.valueList.map((value, i) => {
+      const valueType = typeof value
+      if (Array.isArray(value)) return null // INFO - CH - 20210122 - I currently don't know if this case exists in Json schema spec
+
+      if (valueType === 'object') {
+        return (
+          <div
+            className='DisplaySchemaArray'
+            key={`object_${props.currentKey}_${value.title}_${i}`}
+          >
+            <DisplaySchemaObject
+              schemaObject={props.schemaObject}
+              dataSchemaObject={value}
+              key={`object_${props.currentKey}_${value.title}__DisplaySchemaObject_${i}`}
+            />
+          </div>
+        )
+      }
+
+      return null
+    })
+  ]
+}
+
+const DisplaySchemaObject = props => {
+  if (!props.dataSchemaObject || !props.schemaObject) {
+    console.log('Error in DisplaySchemaObject, null props', props)
+    return null
+  }
+
+  const title = props.schemaObject.title
+    ? (
+      <DisplayTitle
+        label={props.schemaObject.title}
+        key={props.schemaObject.title}
+      />
+    )
+    : null
+
+  return [
+    title,
+    Object.entries(props.dataSchemaObject).map(([key, value], i) => {
+      const valueType = typeof value
+
+      if (Array.isArray(value)) {
+        return (
+          <DisplaySchemaArray
+            label={props.schemaObject.properties[key].title}
+            valueList={value}
+            currentKey={key}
+            schemaObject={props.schemaObject.properties[key].items}
+            key={`array_${key}_${i}`}
```

Comment:
I don't think you should put `i` in the `key` prop. `key should already be unique and we should allow React to match elements if they have the same key.
Also you don't need to use `array_` here. A unique key in the same list is sufficient.

Reply:
I have removed it from DisplaySchemaObject but I have to keep it in DisplaySchemaArray